src/components/sidebar/style.module.css

```diff
@@ -6,7 +6,7 @@
   left: 0;
   overflow-x: hidden;
   overflow-y: auto;
-  padding: var(--sidebar-padding);
+  padding: 24px;
```

Comment:
This was the only place the `sidebar-padding` variable was used so I decided to get rid of it and remove the file it was defined in.